src/frontend/apps/desk/src/features/teams/member-management/components/MemberGrid.tsx

```diff
@@ -104,21 +111,51 @@ export const MemberGrid = ({ team, currentRole }: MemberGridProps) => {
 
   return (
     <>
-      {currentRole !== Role.MEMBER && (
-        <Box $margin={{ all: 'big', bottom: 'small' }} $align="flex-end">
-          <Button
-            aria-label={t('Add members to the team')}
-            style={{
-              width: 'fit-content',
-              minWidth: '8rem',
-              justifyContent: 'center',
-            }}
-            onClick={() => setIsModalMemberOpen(true)}
-          >
-            {t('Add a member')}
-          </Button>
+      <Box
+        $display="flex"
+        $direction="column"
+        $margin={{ all: 'big', bottom: 'small' }}
+        $position="relative"
+      >
+        <Text
+          $theme="primary"
+          $weight="600"
+          $size="1.3rem"
+          $margin={{ bottom: 'big' }}
+        >
+          {t('Group members')}
+        </Text>
+        <Box
+          $display="flex"
+          $direction="row"
+          $justify="space-between"
+          $align="center"
+          $gap="1rem"
+          $css={`
+            & > * {
+              flex: 0.23 0 auto;
+            }
+          `}
```

Comment:
I don't see the diff when I remove it.
```suggestion
```

Reply:
It just widens the input size.

With the CSS rule:

![Screenshot 2024-09-26 at 15 38 10](https://github.com/user-attachments/assets/96e2b1d5-4fd0-4347-943e-89fda339bbb0)

Without it:

![Screenshot 2024-09-26 at 15 38 23](https://github.com/user-attachments/assets/c72cbd27-eb11-45e7-a161-51bb56dc8d20)